pinot-common/src/main/java/com/linkedin/pinot/common/metrics/ServerMeter.java

```diff
@@ -51,7 +51,10 @@
   LLC_CONTROLLER_RESPONSE_COMMIT_SUCCESS("messages", true),
   LLC_CONTROLLER_RESPONSE_COMMIT_CONTINUE("messages", true),
   LLC_CONTROLLER_RESPONSE_PROCESSED("messages", true),
-  LLC_CONTROLLER_RESPONSE_UPLOAD_SUCCESS("messages", true);
+  LLC_CONTROLLER_RESPONSE_UPLOAD_SUCCESS("messages", true),
+  NUM_DOCS_SCANNED("rows", true),
```

Comment:
can we emit these on a per table basis?That can be useful in multi-tenant clusters to check how much load a particular table is causing

Reply:
Done.